style.css

```diff
@@ -579,3 +621,317 @@ label {
+  .front-end2,
+  .desktop-paragraph2,
+  .header-facebook {
+    display: inline;
+  }
+
+  .labels-3 h2 {
+    font-size: 40px;
+    font-weight: 700;
+  }
+
+  .work-three .facebook2 {
+    font-size: 18px;
+    font-family: 'Poppins', sans-serif;
+    line-height: 16px;
+    font-weight: 600;
+    font-style: normal;
+    color: #344563;
+  }
+
+  .work-three .front-end2 {
+    font-size: 18px;
+    color: #7a869a;
+    line-height: 16px;
+    font-weight: 400;
+    font-style: normal;
+    font-family: 'Poppins', sans-serif;
+  }
+
+  .work-three .year {
+    font-size: 18px;
+    font-weight: 400;
+  }
+
+  .desktop-paragraph2 {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .labels-3 .project-button {
+    background-color: #4053fc;
+    color: #fff;
+  }
+
+  .headerz,
+  .uber,
+  .year-1,
+  .desktop-paragraph3,
+  .lead-dev {
+    display: inline;
+  }
+
+  .labels-4 h2 {
+    font-size: 40px;
+  }
+
+  .work-four .uber {
+    font-size: 19px;
+    font-family: 'Poppins', sans-serif;
+    line-height: 16px;
+    font-weight: 600;
+    font-style: normal;
+    color: #344563;
+  }
+
+  .work-four .lead-dev,
+  .work-four .year-1 {
+    font-size: 18px;
+    color: #7a869a;
+    line-height: 16px;
+    font-weight: 400;
+    font-style: normal;
+    font-family: 'Poppins', sans-serif;
+  }
+
+  .desktop-paragraph3 {
+    font-size: 16px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .move {
+    margin-left: 10px;
+  }
+
+  .about-me {
+    width: 100%;
+    display: grid;
+    grid-template-columns: 50% 50%;
+    justify-content: space-evenly;
+    height: auto;
+    margin-top: 10vh;
+    border-top-right-radius: 100px;
+  }
+
+  ul .list-items {
+    width: 120px;
+    height: 120px;
+    display: flex;
+    flex-direction: row;
+  }
+
+  ul .list-items p {
+    position: relative;
+    top: 40px;
+    right: 60px;
+  }
+
+  .list-items1 {
+    position: relative;
+    top: 140px;
+    right: 100px;
+  }
+
+  .list-items2 {
+    position: relative;
+    left: 40px;
+    top: 10px;
+  }
+
+  .list-items3 {
+    position: relative;
+    left: 170px;
+    bottom: 120px;
+  }
+
+  .language {
+    position: relative;
+    top: 150px;
+    right: 45px;
+    font-weight: 500;
+  }
+
+  .frameworks {
+    position: relative;
+    right: 45px;
+    bottom: 100px;
+  }
+
+  .line-1 {
+    position: relative;
+    bottom: 90px;
+    right: 20px;
+  }
+
+  .gained-skill {
+    position: relative;
+    bottom: 90px;
+    right: 38px;
+  }
+
+  .line-2 {
+    position: relative;
+    bottom: 100px;
+    right: 20px;
+  }
+
+  .form-image {
+    display: none;
+  }
+
+  .form-section {
+    background-image: url(images/contact\ form\ .png);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    margin-bottom: 30px;
+  }
```

Comment:
- [ ] Kindly consider removing the `margin-bottom: 30px;` property you have as it's creating an empty negative space at the bottom of the page. ✔
    - Here's what it looks like:
       ![firefox_fFqht2TFjE](https://user-images.githubusercontent.com/104968236/228876842-81a413f4-1e1b-43c6-83f5-092bc5ffdecf.png)